packages/story-editor/src/components/canvas/karma/rightClickMenu.karma.js

```diff
@@ -1017,4 +1058,79 @@ describe('Right Click Menu integration', () => {
       });
     });
   });
+
+  describe('right click menu: multiple elements selected', () => {
+    it('should duplicate all selected elements', async () => {
```

Comment:
💭 this also duplicates animations. It would be good to verify that in this test too.

Reply:
Makes sense as an integration test I guess, but also `duplicateElementById` has test coverage to verify that animations get copied, so not a huge deal either way.

🤔 I guess it's good just from a regression testing perspective if the implementation ever changes